doc/whats_new.rst

```diff
@@ -25,7 +25,12 @@ Authors
 
 * `Alex Rockhill`_
 * `Richard Höchenberger`_
+* `Stefan Appelhoff`_
+* `Diego Lozano-Soldevilla`_
+* `Eduard Ort`_
+* `Alexandre Gramfort`_
```

Comment:
Not sure what happened here, but I didn't actually add any authors other than me